packages/web-components/src/components/overflow-menu/overflow-menu-body.ts

```diff
@@ -131,23 +171,14 @@ class CDSOverflowMenuBody extends CDSFloatingMenu {
 
       if (isInsideMenu) {
         event.preventDefault();
-        this.open = false;
 
-        requestAnimationFrame(() => {
-          const menuTrigger = this.parent as HTMLElement | null;
-          const triggerButton =
-            menuTrigger?.shadowRoot?.querySelector(
-              `button.${prefix}--overflow-menu`
-            ) || menuTrigger?.querySelector(`button.${prefix}--overflow-menu`);
+        const menuTrigger = this.parent as HTMLElement | null;
```

Comment:
Hey @divya-281 , Seeing a regression here - When menu item is selected via Enter key,  the menu closes but focus is lost

https://github.com/user-attachments/assets/dc67988e-e8c4-4c70-8e8b-692b0904ca00



 I think adding back the focus return logic should fix this
 
```suggestion
          requestAnimationFrame(() => {
        const menuTrigger = this.parent as HTMLElement | null;
    const triggerButton =
      menuTrigger?.shadowRoot?.querySelector(
        `button.${prefix}--overflow-menu`
      ) || menuTrigger?.querySelector(`button.${prefix}--overflow-menu`);

    if (triggerButton) {
      (triggerButton as HTMLElement).focus();
    } else {
      console.warn('Could not find trigger button.');
      (document.body as HTMLElement).focus();
    }
  });
```

Reply:
@2nikhiltom When a user selects one of the items in overflow menu the focus does not return back to the trigger button in [react components](https://react.carbondesignsystem.com/?path=/story/components-overflowmenu--default), which is why I removed this, adding this back makes the focus return to the trigger button after user selects an item. Can you let me know what is the expected behaviour?

Reply:
Looking at the Dropdown in both [react](https://web-components.carbondesignsystem.com/?path=/story/components-dropdown--default) and [web-components  ](https://web-components.carbondesignsystem.com/?path=/story/components-dropdown--default) I think the focus should go back to the triggering element after the item is selected or the list is closed via ESC key

Reply:
Thanks for clarifying, I've made the change!